### ISSUE TITLE
Web/upload: broken for view

### ISSUE THREAD

**Issue by p-l-** [cea-sec/ivre#796]
Now the Web UI relies on the `view` purpose, upload no longer work:
```
WARNING:ivre:Could not import /tmp/tmpXXXXXXXX
Traceback (most recent call last):
  File "/home/user/ProjetsPub/ivre/build/lib/ivre/web/app.py", line 532, in import_files
    if subdb.store_scan(fdesc.name, categories=categories,
AttributeError: 'MongoDBView' object has no attribute 'store_scan'
```

**Comment by Flyrabby** [comment]
I discovered this problem several weeks ago.When I debugged this function,I could not step into the attribute 'store_scan'.I thought there are some problems about class Inheritance.So I implemented a function jd2db(json data 2 db) for writing the file of the json data into the views collection of mongodb.And then I am going to use the webui call the jd2db to implement the upload function.
At last,I appreciate the ivre framework very much and feel very sorry for my poor English!

**Comment by p-l-** [comment]
Thanks for your kind words about IVRE :)

The fix (#799) is a bit more complex: uploaded results are expected to be Nmap scan results (rather than "view" documents). So the results are now always uploaded in "nmap", and merged into the "view" when POSTed under `/cgi/view`.
